docs/going-into-production.rst

```diff
@@ -267,19 +276,38 @@ Then, you could configure your data paths like this:
     path.logs: /srv/crate/logs
     path.repo: /srv/crate/snapshots
 
+.. NOTE::
+
+    If you have installed CrateDB using a system package for Debian, Ubuntu, or
+    Red Hat, the `CRATE_HOME`_ variable (as well as some other data paths) are
+    configured for by the `systemd`_ *service* file. View the source code
+    for `Debian`_, `Ubuntu`_, or `Red Hat`_ for the full details.
```

Comment:
The repo is currently not public :/ 

But users can view the definition with `systemctl cat crate`

Reply:
ah! one sec

Reply:
pushed a fixup